docs/install.md

```diff
@@ -206,6 +206,94 @@ Select the following tabs for the installation description of your target.
+
+    The zip file contains a couple of files, the important is the `.vhdx` file which have to be copied to a Hyper-V machine folder (The folder can be located anywhere on the PCs harddisk).
+
+    <font size="+2">2. Add a Hyper-V virtual machine</font>
+
+    As next, a Hyper-V machine is generated. Right click on your PCs node in the left tree and open the dialog wizard for the machine generation ("New" -> "Virtual Machine"):
+
+    ![DietPi Hyper-V machine generation](assets/images/dietpi-HyperV-VM-generation.png){: width="450" height="364" loading="lazy"}
+
+    In the following wizard you have to set the following:
+
+    1. Give your machine a name ("Specify Name and Location")
+    2. Select the Hyper-V Generation: Select **Generation 1** ("Specify Generation")
+    3. Choose your RAM size (e.g. 2048 MB)
+    4. If you have already configured a network, select your network. Otherwise let it "Not connected" and change it afterwards
+    5. Choose to use extracted `.vhdx` Hyper-V disc file (see above)
+
+    If you have not set up any network connection, go on with the **Virtual Switch Manager** and add a network. Select that network in your virtual machine settings afterwards.  
+
+    ![DietPi Hyper-V network management](assets/images/dietpi-HyperV-manage-network.png){: width="250" height="284" loading="lazy"}
+
+    <font size="+2">3. First boot of the new Hyper-V machine</font>
+
+    First, click on ***Connect*** to open a window of the virtual machine:
+
+    ![DietPi Hyper-V open machine windows](assets/images/dietpi-HyperV-connect-machine.png){: width="450" height="205" loading="lazy"}
+
+    Then press ***Start*** to boot up the machine:
+
+    ![DietPi Hyper-V machine start](assets/images/dietpi-HyperV-start-machine.png){: width="550" height="420" loading="lazy"}
+
+    After this, your machine should boot up.
+
+    <font size="+2">Additional information / troubleshooting</font>
+
+    In the case that you did not setup your network configuration properly, the booting procedure will not find a network connection and may respond with this boot console output:
+
+    ![DietPi Hyper-V boot without network](assets/images/dietpi-HyperV-boot-wo-network.jpg){: width="722" height="164" loading="lazy"}
+
+    Then you have to check and repair your network configuration within the **Virtual Switch Manager**.
+
+    It may also be necessary to deselect the "Enable Secure Boot" option in your virtual machine settings (at least select "Microsoft UEFI Certificate Authority"):
+
+    ![DietPi Hyper-V safe start option](assets/images/dietpi-HyperV-safe-start.png){: width="600" height="240" loading="lazy"}
+
+    Another option is to generate your own Hyper-V virtual machine (e.g. Generation 2 with UEFI) using the **UEFI installer image** from the **Native PC** installation like you would do it on a PC. This procedure is not described here.
```

Comment:
While one can use an UEFI image, it is usually on necessary overhead for VMs, as long as one does not want to explicitly test it or wants to use secure boot for some reason, which is a UEFI feature and the reason why it needs to be disabled for the prepared Hyper-V image.
Another issue is that the native PC images come with a lot of firmware and drivers pre-installed that are obsolete on VMs. And due to the hardware ID, our scripts will detect the system as native PC rather than as VM, and will show and/or miss a few configs and install options because of that.

So in our official documentation we should not mention that idea, even that it generally works. If there is enough demand for UEFI VM images, we should better created such explicitly with packages and hardware ID as we expect and intend it for VMs.

But one clean option that could be mentioned here as alternative, is to use an official Debian image, or the Debian installer and run DietPi-PREP to create a custom UEFI VM image. For x86_64 systems of all kinds that works pretty well.

Generally a Debian installer guide (on a separate docs page) would be nice by times, like here but with pictures 🙂: https://github.com/MichaIng/DietPi/wiki/How-to-create-a-DietPi-image-for-x86_64-PCs-(UEFI)
That becomes more relevant with Debian Bullseye as many SBCs are now supported as well natively by it.

Reply:
Aaah, I understand. I will kick it out.

I will test to generate an image with a Debian netinstaller within Hyper-V and afterwards do a DietPi-PREP and look how this works. Then I can add a small section about this.

We could enter an issue regarding the wiki article with bells and whistles. :laughing:

Reply:
#377 😃

But it should be seen as low priority, as this goes beyond of what most users would want to go through 😉.

Reply:
Image generation with debian netinst.iso within Hyper-V (Generation 2) and doing DietPi-PREP afterwards worked. No deep testing was done for now.  
But it shows that this might be an option for the generation of a Hyper-V generation 2 machine with DietPi-PREP. I will add this as a short description.